arch/x86/kernel/cpu/resctrl/monitor.c

```diff
@@ -710,10 +722,11 @@ static int dom_data_init(struct rdt_resource *r)
 	}
 
 	/*
-	 * RMID 0 is special and is always allocated. It's used for all
-	 * tasks that are not monitored.
+	 * CLOSID 0 and RMID 0 are special and are always allocated. These are
+	 * used for rdtgroup_default control group, which will be setup later.
+	 * See rdtgroup_setup_root().
 	 */
-	entry = __rmid_entry(0);
+	entry = __rmid_entry(0, 0);
```

Comment:
At upstream instead of using 0 they use variables. The variables that you have defined in the next commit.

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev